cmd/prometheus/main.go

```diff
@@ -1193,6 +1203,14 @@ func (s *readyStorage) Stats(statsByLabelName string) (*tsdb.Stats, error) {
 	return nil, tsdb.ErrNotReady
 }
 
+// WALReplayStatus implements the api_v1.TSDBStats interface.
```

Comment:
```suggestion
// WALReplayStatus implements the github.com/prometheus/prometheus/web/api/v1.TSDBStats interface.
```

Reply:
The other function implementations don't though

Reply:
Nothing is perfect, feel free to ignore this then or fix the other too (: